File: SE4_C#/EJ4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE4_C_
{
    internal class EJ4
    {
        static void Main(string[] args)
        {
            double area;


            Console.WriteLine("------SISTEMA DE CÁLCULO DE ÁREAS------");
            Console.WriteLine("\n(1) Área de un cuadrado.");
            Console.WriteLine("(2) Área de un rectángulo.");
            Console.WriteLine("(3) Área de un triángulo.");
            Console.WriteLine("(4) Área de un círculo.\n");
            int opcion = int.Parse(Console.ReadLine());


            switch (opcion) 
            {
                case 1:
                    Console.WriteLine("Ingrese el lado del cuadrado: ");
                    double L = double.Parse(Console.ReadLine());
                    area = L * L;
                    Console.WriteLine($"\nEl área del cuadrado es de: {area} metros cuadrados.");
                    break;
                case 2:
                    Console.WriteLine("Ingrese la base del rectángulo: ");
                    double b = double.Parse(Console.ReadLine());

                    Console.WriteLine("Ingrese la altura del rectángulo: ");
                    double a = double.Parse(Console.ReadLine());

                    area = b * a;
                    Console.WriteLine($"\nEl área del rectángulo es de: {area} metros cuadrados.");
                    break;
                case 3:
                    Console.WriteLine("Ingrese la base del triángulo: ");
                    double bt = double.Parse(Console.ReadLine());

                    Console.WriteLine("Ingrese la altura del triángulo: ");
                    double at = double.Parse(Console.ReadLine());

                    area = (bt * at)/2;
                    Console.WriteLine($"\nEl área del triángulo es de: {area} metros cuadrados.");
                    break;
                case 4:
                    Console.WriteLine("Ingrese el radio del círculo: ");
                    double r = double.Parse(Console.ReadLine());
                    area = Math.PI * Math.Pow(r, 2); ;
                    Console.WriteLine($"\nEl área del círculo es de: {area} metros cuadrados.");
                    break;
                default:
                    Console.WriteLine("Ha ingresado una opción incorrecta.");
                    break;

            }
        }


    }
}

File: SE4_P/ej4.py
import math

print("------SISTEMA DE CÁLCULO DE ÁREAS------")
print("\n(1) Área de un cuadrado.")
print("(2) Área de un rectángulo.")
print("(3) Área de un triángulo.")
print("(4) Área de un círculo.\n")
opcion = int(input("Elija una opción: "))


match (opcion):
    case 1:
        L = int(input("Ingrese el lado del cuadrado: "))
        area = L * L
        print(f"\nEl área del cuadrado es de: {area} metros cuadrados.")
    case 2:
        b = int(input("Ingrese la base del rectángulo: "))
        a = int(input("Ingrese la altura del rectángulo: "))
        area = b * a
        print(f"\nEl área del rectángulo es de: {area} metros cuadrados.")
    case 3:
        bt = float(input("Ingrese la base del triángulo: "))
        at=float(input("Ingrese la altura del triángulo: "))
        area = (bt * at)/2
        print(f"\nEl área del triángulo es de: {area} metros cuadrados.")
    case 4:
        r=float(input("Ingrese el radio del círculo: "))
        area = math.pi * math.pow(r, 2); 
        print(f"\nEl área del círculo es de: {area} metros cuadrados.")
    case default:
        print("Ha ingresado una opción incorrecta.")

